### ISSUE TITLE
🐛 Bug: Logo color is incorrect on initial load in dark mode

### ISSUE THREAD

**Issue by AdityaBhate** [json-schema-org/website#756]
### Describe the bug

When visiting the website for the first time, the default theme is dark mode, but the logo color is black instead of white. This issue is resolved when toggling the theme switcher (light/dark mode).

### Steps To Reproduce

1. Visit the website for the first time (ensure no previous theme settings are stored).
2. Observe the logo color in dark mode.
3. Toggle the theme switcher to light mode and back to dark mode.
4. Observe the logo color changes to white correctly.

### Expected Behavior

The logo should be white on the initial load when dark mode is the default theme.

### Screenshots

Initial Load (chrome, incognito)
![image](https://github.com/json-schema-org/website/assets/98549181/8b310447-d32c-45b8-9e8f-8cbe9d81b806)

---

After Toggling the to Light Mode and then to Dark Mode
![ezgif-5-6862d1b9d8](https://github.com/json-schema-org/website/assets/98549181/614b1717-46dd-4c96-8917-c0971a9c18b1)

### Device Information [optional]

```markdown
- OS: Windows 11
- Browser: Chrome
- version: 125.0.6422.142 (Official Build) (64-bit)

- Browser: Firefox
- version: Version 126.0.1 (64-bit)
```

### Are you working on this issue?

Yes

### Do you think this work might require an [Architectural Decision Record (ADR)]? (significant or noteworthy)

Yes

**Comment by AdityaBhate** [comment]
Assign me this issue, I would like to work on this

**Comment by JeelRajodiya** [comment]
Hi @AdityaBhate. This issues is resolved in pull request #735
